tests/system/feature_store/test_feature_store.py

```diff
@@ -2454,6 +2454,18 @@ def test_deploy_ingestion_service_with_different_targets(
         for expected in expected_target_names:
             assert fset.get_target_path(expected) is not None
 
+    def test_feature_vector_with_all_features_and_label_feature(self):
+        feature_set = FeatureSet("fs-label", entities=[Entity("ticker")])
+        fs.ingest(feature_set, stocks)
+        expected = stocks.to_dict()
+        expected.pop("ticker")
+
+        fv = fs.FeatureVector("fv-label", ["fs-label.*"], "fs-label.name")
+        res = fs.get_offline_features(fv)
+
+        assert res is not None
+        assert res.to_dataframe().to_dict() == expected
+
```

Comment:
Add a test to cover the 2 fsets with the same feature name scenario. I do hope it works 😄